docs/src/integrations/exchange.md

```diff
@@ -7,9 +7,23 @@ exchange.
 
 ## Node Setup
 
-We highly recommend setting up at least two of your own Solana api nodes to
-give you a trusted entrypoint to the network, allow you full control over how
-much data is retained, and ensure you do not miss any data if one node fails.
+This section explains various setup topics to run the Solana full node (called
+`solana-validator`).
+
+In short, we highly recommend setting up at least two nodes on high-grade
```

Comment:
NIT: "Never go to sea with two chronographs.  Take one or three." :wink:

Reply:
heh. Our chronometers can tell it's broken or not on its own. That's a very fundamental operating property for it. :)